server/utils/utils.ts

```diff
@@ -26,4 +27,12 @@ const initialiseName = (fullName?: string): string | null => {
 
 const hasRole = (user: Express.User, role: AuthRole): boolean => user?.userRoles.includes(role) || false
 
-export { convertToTitleCase, initialiseName, hasRole }
+const parseDate = (date: string) => {
```

Comment:
**issue**: we should be receiving iso dates YYYY-MM-DD.
better name `parseIsoDate`  ?

Reply:
we are receiving this [dd/MM/yyyy](https://github.com/ministryofjustice/hmpps-assess-for-early-release-api/pull/14/files#diff-84a4d63fbf1fda5ae236e5c9cfae37374ec57be3d019829f9d97cfd2b2f85c4fR21) from backend API.
Ya will rename it.

Reply:
we shouldn't be - we can correct that